droid-results/src/main/java/uk/gov/nationalarchives/droid/profile/CsvItemWriter.java

```diff
@@ -220,7 +256,43 @@ public void open(final Writer writer) {
         if (headers == null) {
             headers = Arrays.copyOf(HEADERS, HEADERS.length) ;
         }
-        String[] headersToWrite = getHeadersToWrite(headers);
+    }
+
+    private void writeHeadersForOneLinePerFormatExport(List<? extends ProfileResourceNode> nodes) {
```

Comment:
Is this function name correct, should it not be `writeHeadersForOneRowPerFileExport`? My understanding is that if a file has >1 formats this function will add up to 4 extra columns for each additional format but each file will still only have 1 row.

Reply:
That's correct. I'll rename